broker/core/express-commons/src/middleware/index.js

```diff
@@ -35,6 +36,35 @@ exports.basicAuth = function (username, password) {
   };
 };
 
+exports.tlsAuth = function () {
+  return function (req, res, next) {
+    const clientCertificate = req.headers['ssl-client-cert'];
+    if (!clientCertificate) {
+      logger.error('clientCertificate not found in the request headers: ', req.headers);
+      next(new Unauthorized());
+    } else {
+      const smCertSubjectPattern = config.smConnectionSettings.sm_certificate_subject_pattern;
```

Comment:
```suggestion
      const smCertSubjectPattern = _.get(config, 'smConnectionSettings.sm_certificate_subject_pattern');
```
I think there may be chances that `smConnectionSettings` might be commented out in the settings.yml which may lead to panic. `lodash.get` takes care of this.